monkey/infection_monkey/master/control_channel.py

```diff
@@ -19,37 +19,51 @@ def __init__(self, server: str, agent_id: str):
         self._control_channel_server = server
 
     def should_agent_stop(self) -> bool:
-        if not self._control_channel_server:
-            return
-
```

Comment:
What does this have to do with credentials in config?

Reply:
Nothing, but it's a minor fix that I don't think distracts from the purpose of this PR.